Code/Editor/EditorViewportSettings.cpp

```diff
@@ -491,6 +492,16 @@ namespace SandboxEditor
         AzToolsFramework::SetRegistry(CameraDefaultOrbitDistanceSetting, distance);
     }
 
+    bool CameraGoToPositionInstantly()
```

Comment:
I know you hate putting Get in the name of getters, but this does sound like it could be doing something.

Reply:
Hmm fair point... I'd rather not add `Get` as we don't use it for any other functions in this file, what about `CameraGoToPositionInstantlyEnabled` which fits with the other naming in the file? I'll update the name now.